content/en/docs/reference/kubernetes-api/common-definitions/quantity.md

```diff
@@ -30,27 +30,33 @@ Quantity is a fixed-point representation of a number. It provides convenient mar
 
 The serialization format is:
 
-\<quantity>        ::= \<signedNumber>\<suffix>
-  (Note that \<suffix> may be empty, from the "" case in \<decimalSI>.)
+``` \<quantity>        ::= \<signedNumber>\<suffix>
+
+	(Note that \<suffix> may be empty, from the "" case in \<decimalSI>.)
+
 \<digit>           ::= 0 | 1 | ... | 9 \<digits>          ::= \<digit> | \<digit>\<digits> \<number>          ::= \<digits> | \<digits>.\<digits> | \<digits>. | .\<digits> \<sign>            ::= "+" | "-" \<signedNumber>    ::= \<number> | \<sign>\<number> \<suffix>          ::= \<binarySI> | \<decimalExponent> | \<decimalSI> \<binarySI>        ::= Ki | Mi | Gi | Ti | Pi | Ei
-  (International System of units; See: http://physics.nist.gov/cuu/Units/binary.html)
+
+	(International System of units; See: http://physics.nist.gov/cuu/Units/binary.html)
+
 \<decimalSI>       ::= m | "" | k | M | G | T | P | E
-  (Note that 1024 = 1Ki but 1000 = 1k; I didn't choose the capitalization.)
-\<decimalExponent> ::= "e" \<signedNumber> | "E" \<signedNumber>
+
+	(Note that 1024 = 1Ki but 1000 = 1k; I didn't choose the capitalization.)
+
+\<decimalExponent> ::= "e" \<signedNumber> | "E" \<signedNumber> ```
 
 No matter which of the three exponent forms is used, no quantity may represent a number greater than 2^63-1 in magnitude, nor may it have more than 3 decimal places. Numbers larger or more precise will be capped or rounded up. (E.g.: 0.1m will rounded up to 1m.) This may be extended in the future if we require larger or smaller quantities.
 
 When a Quantity is parsed from a string, it will remember the type of suffix it had, and will use the same type again when it is serialized.
 
 Before serializing, Quantity will be put in "canonical form". This means that Exponent/suffix will be adjusted up or down (with a corresponding increase or decrease in Mantissa) such that:
-  a. No precision is lost
-  b. No fractional digits will be emitted
-  c. The exponent (or suffix) is as large as possible.
+
+- No precision is lost - No fractional digits will be emitted - The exponent (or suffix) is as large as possible.
```

Comment:
BTW we're turning a list into not-a-list here.

Reply:
This is due to the changes in the doc of the original source code. I'm not sure we can do anything here.

We can also see the side effect of the changes done for the Quantity definition, adding triple quotes around the definition.

The problem is that the doc lines are all concatenated into a single line to create the definition in the swagger file, adding \n's when blank lines are found, but no other markdown are respected.

Reply:
related commit on source code: https://github.com/kubernetes/kubernetes/commit/971046e26e830f9b53219c7b0a078213b338572c

Reply:
OK, that's a clear explanation.